.github/workflows/meta-balena-esr.yml

```diff
@@ -55,6 +55,7 @@ jobs:
           # The default GITHUB_TOKEN does not have workflow scope
           # This is needed to push a new branch with workflow files
           token: ${{ steps.gh_app_token.outputs.token }}
+          persist-credentials: true
```

Comment:
This is just so we can easily find which repos are using the insecure method, correct?

Reply:
Yes it lets us find them and also makes it clear to everyone looking at the workflow that it's the case, basically raising awareness